com.salesforce.b2eclipse.jdt.ls/src/main/java/com/salesforce/b2eclipse/command/BazelWorkspaceCommandRunner.java

```diff
@@ -466,6 +508,19 @@ public void runBazelVersionCheck() throws BazelCommandLineToolConfigurationExcep
         }
     }
 
+    public String getOperatingSystemDirectoryName() {
+        if (SystemUtils.IS_OS_MAC) {
+            return "darwin";
+        }
+        if (SystemUtils.IS_OS_WINDOWS) {
+            return "windows";
+        }
+        if (SystemUtils.IS_OS_LINUX) {
+            return "linux";
```

Comment:
Isn't it `k8` sometimes?

Reply:
it should be as a part of output path, not OS-dependent